packages/fast-components-react-msft/src/divider/__snapshots__/divider.spec.ts.snap

```diff
@@ -3,6 +3,7 @@
 exports[`divider divider: 0 1`] = `
 <hr
   className="divider-0-1-1"
+  data-sketch-symbol="Divider"
```

Comment:
Is it possible that the "data-sketch-symbol" value could be derived from the component that the script is executed on in a way to eliminate the need for this attribute altogether?

Reply:
I don't think it would give us the clarity we want. This value is harvested and applied as a Symbol name in sketch (and also used a selector to find sketch symbol sources), and right now we don't have the consistency between component values to do this accurately.